README.md

```diff
@@ -459,6 +462,7 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
 
 - [Deep Learning for NLP, advancements and trends in 2017](https://tryolabs.com/blog/2017/12/12/deep-learning-for-nlp-advancements-and-trends-in-2017/?utm_campaign=Data%2BElixir&utm_medium=email&utm_source=Data_Elixir_162)
 - [10 Exciting Ideas of 2018 in NLP](http://ruder.io/10-exciting-ideas-of-2018-in-nlp/)
+- [Repository of NLP Ressources](https://github.com/stepthom/text_mining_resources)
```

Comment:
I appreciate the humor! But let's save some poor soul a few hours of infinite looping.

Reply:
expected, but disappointing none the less, 
removed